eduaid_web/src/App.css

```
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap');
```

Stylelint (17.5.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap'" (import-notation)

(import-notation)